spell/mom-onboarding-checklist.md

```diff
@@ -0,0 +1,30 @@
+# Mom Onboarding Checklist
+  * [ ] Deployed Contracts
+    * [ ] `Mom`
+      * [ ] deployed via deployer
+        * [ ] ensure different deployer is used for mainnet and testnet to avoid contracts with same address and different sources
+        * [ ] contract is verified on etherscan
+          * [ ] ensure solc version matches source
+          * [ ] ensure optimization matches source
+          * [ ] ensure license `AGPLv3` is specified
+          * [ ] ensure source matches github code (i.e. diffcheck via vscode `code --diff etherscan.sol github.sol`)
+        * [ ] constructor args are correct
+        * [ ] check `owner`
+          * [ ] `MCD_PAUSE_PROXY` is set as `owner` via `setOwner`
```

Comment:
should this also specify that the deployer is not authorized anymore?

Reply:
hmm, yeah, it should probably be specified, good point.

Reply:
I've added a comment in 104b4db, let me know if that looks good.